package.json

```diff
@@ -40,13 +40,13 @@
   "dependencies": {
     "body-parser": "^1.6.5",
     "express": "4.x",
-    "grunt-mocha": "0.4.x",
+    "grunt-lib-phantomjs": "^0.7.0",
     "istanbul": "^0.3.0",
     "jade": "1.x",
     "lodash": "^2.4.1",
-    "moment": "^2.8.1",
     "minimatch": "^2.0.4",
-    "phantomjs": "1.9.13",
```

Comment:
Don't remove this line.  Newer versions of phantomjs fail to install nearly 25% of the time in automated Jenkins jobs.  The workaround is to force the use of this version of phantomjs (there's a shrinkwrap.json for this reason also).